modules/images-configuration-file-hcp.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-configuration-file-hcp_{context}"]
+= Configuring image registry settings for HCP
+
+You can configure image registry settings by editing the `image.config.openshift.io/cluster` custom resource (CR).
+When changes to the registry are applied to the `image.config.openshift.io/cluster` CR, the Machine Config Operator (MCO) performs the following sequential actions:
+
+. Cordons the node
+. Applies changes by restarting CRI-O
+. Uncordons the node
++
+[NOTE]
+====
+The MCO does not restart nodes when it detects changes.
+====
+
+.Procedure
+
+. Edit the `image.config.openshift.io/cluster` custom resource:
```

Comment:
This page pretty much need a rewrite. Replacement will be rosa commands for create, edit and describe.